fabric-object-builder-api-v1/src/testmod/java/net/fabricmc/fabric/test/object/builder/ObjectBuilderTestConstants.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.test.object.builder;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import net.minecraft.util.Identifier;
+
+public final class ObjectBuilderTestConstants {
+	public static final String MOD_ID = "fabric-object-builder-api-v1-testmod";
+	public static final Logger LOGGER = LogManager.getLogger(MOD_ID);
+
+	public static Identifier id(String name) {
+		return new Identifier(MOD_ID, name);
+	}
+}
```

Comment:
Why not make all of these constants and the logger part of the ModInitializer for the test mod

Reply:
Because we will have other entrypoints testing different parts of object builder in the future

Reply:
Hmm yeah that makes sense